lib/compiler_rt/gehf2.zig

```diff
@@ -22,10 +27,10 @@ pub fn __gthf2(a: f16, b: f16) callconv(.C) i32 {
     return __gehf2(a, b);
 }
 
-fn __aeabi_fcmpge(a: f16, b: f16) callconv(.AAPCS) i32 {
+fn __aeabi_hcmpge(a: f16, b: f16) callconv(.AAPCS) i32 {
```

Comment:
If I search for fcmpge, then I get for example this: https://github.com/google/google-ctf/blob/master/third_party/edk2/ArmPkg/Library/ArmSoftFloatLib/Arm/__aeabi_fcmpge.c
However, I can not find anything for hcmpge. May I ask, which use case/vendor lib requires __aeabi_hcmpge ?